README.md

```diff
@@ -16,7 +16,7 @@ The SDK expects the OpenRVS code (both `OpenRVS` and `OpenRenderFix` directories
 1. Run `"SDK ToolBelt.bat"` to enter the CLI
 1. Run `160` to activate the SDK for Raven Shield v1.60
 1. Type `compile` to enter the compiler, and then type `OpenRVS` to compile OpenRVS. Address any errors or warnings if necessary. Type `log` to show more detail
-1. Type `strip` to enter the symbol/docstring stripper, and then type `OpenRVS` to run it
+1. Type `strip` to enter the symbol/docstring stripper, and then type `OpenRVS` to run it. This will reduce output file size by around 60%
```

Comment:
If I recall you need to include file extension with the strip commandlet.  So this needs to change to "type 'OpenRVS.u' "